src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -220,6 +220,15 @@ export default class DeclarativeTableComponent extends ContainerBase<any, azdata
 		return '';
 	}
 
+	public getCheckAllColumnAriaLabel(colIdx: number): string {
+		return localize('checkAllColumnLabel', "check all {0}", this.columns[colIdx].displayName);
```

Comment:
1. the text is not very clear: check all checkboxes in column: {0}
1. checkbox does have state, it should be 'check ....' or 'uncheck...'